src/features/unified-share-modal/messages.js

```diff
@@ -203,31 +203,21 @@ const messages = defineMessages({
         description: 'Label for a shared link permission level',
         id: 'boxui.unifiedShare.sharedLinkPermissionsViewDownload',
     },
-    sharedLinkPermissionsViewDownloadDescription: {
-        defaultMessage: 'Users can view and download',
-        description: 'Description for Can view and download option',
-        id: 'boxui.unifiedShare.sharedLinkPermissionsViewDownloadDescription',
-    },
     sharedLinkPermissionsViewOnly: {
         defaultMessage: 'Can view only',
         description: 'Label for a shared link permission level',
         id: 'boxui.unifiedShare.sharedLinkPermissionsViewOnly',
     },
     sharedLinkPermissionsEdit: {
         defaultMessage: 'Can edit',
-        description: 'Label for a shared link permission to show for editable box notes',
+        description: 'Label for a shared link permission to show for an editable box note / file',
```

Comment:
Since we already have an existing message for the label 'Can Edit', I opted to change the description instead rather than duplicate this message into a File / Box Note specific message.